_scripts/make.go

```diff
@@ -52,9 +53,11 @@ func NewMakeCommands() *cobra.Command {
 		Run: func(cmd *cobra.Command, args []string) {
 			tagFlag := prepareMacnative()
 			execute("go", "install", tagFlag, buildFlags(), DelveMainPackagePath)
+			dlvExecutablePath := installedExecutablePath()
 			if runtime.GOOS == "darwin" && os.Getenv("CERT") != "" && canMacnative() {
-				codesign(installedExecutablePath())
+				codesign(dlvExecutablePath)
 			}
+			copyDAPBinary(dlvExecutablePath)
```

Comment:
I think this should only be done if a flag is passed to the install command, like `go run _scripts/make.go install -dap`. This thing is very specific to VSCode-go and Delve is used in other contexts too.

Reply:
PS: This is the way I found to do it since I'm a bit of a novice to makefiles, if you know a better solution I'd appreciate it

Reply:
VSCode Go's installing dlv@master as `dlv-dap` is a temporary workaround which we plan to get rid of as soon as dlv dap becomes relatively stable (at that point, we probably want to install one single 'dlv' binary used for both legacy/dlv-dap adapter modes). 

You won't need to use make to install your dev version of dlv. From the delve project directory,

go build -o <path_to_dlv_dap_dest> ./cmd/dlv